### PR TITLE
Router

### PR DESCRIPTION
**Pull request by Casey1449**
- Added routes with 'Home' component as index route
- Changed Blake's buttons from 'sections' to 'Links' to add routing.

### CODE REVIEW

**Comment by Casey1449** [comment]
Remember to `npm i` after pulling to install mike's data viz library. It's starting to look like an app!

**Comment by Casey1449** [comment]
closes #19

File: main.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {app, BrowserWindow} from 'electron';
 import menubar from 'menubar';
 
-let mb = menubar({maxWidth: 300, maxHeight: 300, minWidth: 300, minHeight: 300, alwaysOnTop: true});
+let mb = menubar({maxWidth: 800, maxHeight: 800, minWidth: 300, minHeight: 300, alwaysOnTop: true});
```
Comment (blakeworsley) [inline main.js:4]: still working on getting this to be an interchangeable size but I think this is good for dev.

**Comment by kylem038** [comment]
This looks good 🚢 🇮🇹

**Comment by blakeworsley** [comment]
Looks good Casey!

**Pull request merged**